contxt/services/contxt_deployments.py

```diff
@@ -0,0 +1,50 @@
+from typing import List, Optional
+
+from ..auth import Auth
+from ..models.contxt import Cluster, ServiceGrant
+from .api import ApiEnvironment, ConfiguredApi
+
+
+class ContxtDeploymentService(ConfiguredApi):
+    """Contxt API client"""
+
+    _envs = (
+        ApiEnvironment(
+            name="production",
+            base_url="https://contxt.api.ndustrial.io/deploy/v1",
+            client_id="8qY2xJob1JAxhmVhIDLCNnGriTM9bct8",
+        ),
+    )
+
+    def __init__(self, auth: Auth, env: str = "production", **kwargs) -> None:
+        super().__init__(env=env, auth=auth, **kwargs)
+
+    """
+    Clusters
+    """
+
+    def get_clusters(self, organization_id: str) -> List[Cluster]:
+        return [Cluster.from_api(rec) for rec in self.get(f"{organization_id}/clusters")]
+
+    def get_cluster(self, organization_id: str, cluster_slug: str) -> Cluster:
+        resp = self.get(f"{organization_id}/clusters/{cluster_slug}")
+        return Cluster.from_api(resp)
+
+    def register_cluster(
```

Comment:
I normally would agree but I think in this case it's more accurate to call it register as the "Create" action of a cluster might get confusing since we're not actually creating an actual cluster, but rather registering that it does exist.